app/lib/staypuft/seeder.rb

```diff
@@ -411,8 +411,8 @@ def functional_dependencies
       cisco_nexus_config             = { :hash => '<%= n = @host.deployment.neutron; (n.active? && n.cisco_nexus_mechanism?) ? n.compute_cisco_nexus_config : {} %>' }
 
       # Cisco N1KV params
-      n1kv_vsm_ip                    = { :string => '<%= n = @host.deployment.neutron; (n.active? && n.n1kv_plugin?) ? n.n1kv_vsm_ip : "" %>' }
-      n1kv_vsm_password              = { :string => '<%= n = @host.deployment.neutron; (n.active? && n.n1kv_plugin?) ? n.n1kv_vsm_password : "" %>' }
+      n1kv_vsm_ip                    = { :string => '<%= n = @host.deployment.neutron; (n.active? && (n.n1kv_plugin? || n.cisco_n1kv_mechanism?)) ? n.n1kv_vsm_ip : "" %>' }
+      n1kv_vsm_password              = { :string => '<%= n = @host.deployment.neutron; (n.active? && (n.n1kv_plugin? || n.cisco_n1kv_mechanism?)) ? n.n1kv_vsm_password : "" %>' }
```

Comment:
So from this it looks like we're not adding any new params here -- we're simply setting the already-configured-somewhere n1kv_vsm_ip and _password when this new checkbox is selected. If that is what's desired, then this looks good to me.